### PR TITLE
[eclipse/xtext#1569] reduced usage of xtend in purexbase

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1569] reduced usage of xtend in purexbase
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java [hunk 1]
```diff
@@ -8,8 +8,8 @@
 /**
  * Provides labels for IEObjectDescriptions and IResourceDescriptions.
  * 
- * See https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
+ * https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
```
Comment (kthoms) [inline org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java]: Why remove "See" here? Maybe even use `@see`?

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/PureXbaseUiModule.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Use this class to register components to be used within the Eclipse IDE.
+ */
+public class PureXbaseUiModule extends AbstractPureXbaseUiModule {
+
+	public static class NoFolding implements IFoldingRegionProvider {
+		@Override
+		public Collection<FoldedPosition> getFoldingRegions(IXtextDocument xtextDocument) {
+			return Collections.<FoldedPosition> emptySet();
```
Comment (kthoms) [inline org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/PureXbaseUiModule.java]: static import? 
`emptySet()` should be enough

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * generated by Xtext
+ */
+package org.eclipse.xtext.purexbase.ui.quickfix;
+
+import org.eclipse.xtext.xbase.ui.quickfix.XbaseQuickfixProvider;
+
+/**
+ * Custom quickfixes.
+ *
+ * See https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#quick-fixes
```
Comment (kthoms) [inline org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java]: `@see` ?

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java]: this is what the generator "generates"

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java (hunk 3 above)
Comment (cdietrich) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java]: this is what the generator generates

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/labeling/PureXbaseDescriptionLabelProvider.java]: this is what the generator generates

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java (hunk 3 above)
Comment (cdietrich) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java]: this is what the generator generates

**Comment by cdietrich** [comment]
@kthoms i have created https://github.com/eclipse/xtext-core/issues/1272 about the generator

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java (hunk 3 above)
Comment (kthoms) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/quickfix/PureXbaseQuickfixProvider.java]: OK

File: org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/PureXbaseUiModule.java (hunk 2 above)
Comment (kthoms) [reply org.eclipse.xtext.purexbase.ui/src/org/eclipse/xtext/purexbase/ui/PureXbaseUiModule.java]: This is just cosmetics. I would be fine with leaving this. Either change it or leave it.

**Pull request merged**